syntax = "proto3";
import "InstancedItem.proto";
import "StackableItem.proto";

message Inventory {
	FullInventory full = 1;
}

message FullInventory {
	repeated StackableItem stackables = 1;
	repeated InstancedItem instanced = 2;
	uint32 version = 3;
}